doc/intro.rst

```diff
@@ -40,6 +40,13 @@ will send notification to project subscribers if the test result metrics exceed
 a certain value. The threshold values will also appear in the charts. Projects 
 have the `project_settings` field for any specific configuration it might require.
 
+Builds can be compared against others, exposing regressions and fixes. Visit
```

Comment:
I propose this wording: "Builds can be compared against baselines exposing regressions and fixes."

Reply:
Agree, "baseline" is more abstract